src/client/scripts/exportRoom.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+/* global Cards:false Rooms:true*/
+
+Template.exportRoom.helpers({
+
+    getCards : function(category) {
+        var mongoQuery = {
+            "roomCode": Session.get("roomNumber"),
+            "category": category
+        };
+
+        return Meteor.apply("getCardCollectionByQuery",
```

Comment:
{ returnStubValue: true }.  From my understanding, the client only stubs of the meteor methods as the actually code resides on the server.  This tells the client that the stub is going to return a value and that value needs to be returned. This solves the issue where you make a call to the the server and get undefined returned due to the async nature of meteor/node.

Bottom line, apply works, call doesn't.